### ISSUE TITLE
Minecraft client keeps crashing and producing the same error report

### ISSUE THREAD

**Issue by GrimJellynton** [xJon/The-1.7.10-Pack#1321]
I have recently started playing on a friend's server that is running your mod pack. Every so often my client completely crashes and I have to restart it which can take some time. The crash reports produced are identical to one another and an example of one is attached to the post.

The server is running 0.9.3 of the modpack and we have added the following mods:

Aether
AppleCore
Archimedes Ships 
Custom NPCs
Decocraft
DrZarks MosCreatures
ExtraTic
Gilded games util
Open Modular Turrets
Spice of Life

And removed the following for the stated reasons:

RFTools (interfered with the Aether dimension causing it to become collapsed) 
MCA (caused a number of client crashes)
RadixCore (related to MCA)

[crash-2017-01-12_23.20.36-client.txt](https://github.com/xJon/The-1.7.10-Pack/files/703149/crash-2017-01-12_23.20.36-client.txt)

**Comment by xJon** [comment]
I'm sorry but this is a Fastcraft crash and I can't provide you support due to all of the additional mods.
You should post the crash log [there](http://forum.industrial-craft.net/index.php?page=Thread&threadID=10820).
Good luck!

**Comment by Curunir** [comment]
Modifying your local installation is generally not supported. You can do it, but don't expect us to look into the problems you are creating that way.
That being said, here is some free advice: You need to make sure that the server and all clients run the exact same set of mods. If they do not, Forge will usually throw a fit on connect, but it can also lead to strange crashes. Whenever your game consistently crashes, try to start a new singleplayer world, to see if your current mod selection can work together in principle. If it crashes also there, you basically broke your pack. If the new world works, then something in the current world is causing the problem and needs to be removed. Good luck finding it, maybe the crash logs can give you coordinates.

**Comment by GrimJellynton** [comment]
Thank you for the quick reply and advice! I'll pass it onto the admin and continue investigating. If I find a solution, I'll let you know.

**Comment by GrimJellynton** [comment]
We've discovered the solution to our crashes. I contacted a developer of the FastCraft mod and they said a mod that's part of Aether and Gilded Games called mod_ThreadedLighting is "inherently broken" and is not supported by FastCraft. We have completely removed Aether and Gilded Games from the server and this has stopped the frequent client crashes.

Thank you everyone with your help on this matter!
